engine/escalationmanager/db.go

```diff
@@ -82,6 +83,12 @@ func NewDB(ctx context.Context, db *sql.DB, log *alertlog.Store) (*DB, error) {
 			returning ep_step_id, user_id
 		`),
 
+		cleanupMaintenanceMode: p.P(`
+				update services s
+				set maintenance_expires_at = null
+				where s.maintenance_expires_at notnull and now() > s.maintenance_expires_at
```

Comment:
This can be simplified as `>` already requires `notnull` and should be `>=` since it express *at*

Also, a nit, but `table_row` - `comparator` - `value` order would be more consistent with the rest of the module.
```suggestion
				where s.maintenance_expires_at <= now()
```